client/cli/src/client_proxy.rs

```diff
@@ -113,13 +113,14 @@ impl ClientProxy {
     pub fn new(
         host: &str,
         ac_port: u16,
+        json_rpc_port: u16,
         faucet_account_file: &str,
         sync_on_wallet_recovery: bool,
         faucet_server: Option<String>,
         mnemonic_file: Option<String>,
         waypoint: Option<Waypoint>,
     ) -> Result<Self> {
-        let mut client = GRPCClient::new(host, ac_port, waypoint)?;
+        let mut client = GRPCClient::new(host, ac_port, json_rpc_port, waypoint)?;
```

Comment:
shall we rename this to something like AdmissionControlClient  ? (Or LibraClient)